src/dropdown/index.ts

```diff
@@ -413,6 +416,7 @@ let exportDropdown;
       this._buttonEl.focus();
       this._selectorVisible = true;
 
+      if (this.items.length == 0) return;
```

Comment:
1. Please fix the warnings

2. I think it's enough to use this judgment code here.
Other judgment code can be removed.

3. Please move it between "this._buttonEl.focus();" and "this._selectorVisible = true;"